server/controllers/role.js

```diff
@@ -0,0 +1,49 @@
+/* eslint no-underscore-dangle: "off" */
+/* eslint no-param-reassign: "off" */
+/* eslint consistent-return: "off" */
+/* eslint no-shadow: "off"*/
```

Comment:
Instead of including this rules in so many files you could add them to your `.eslintrc` file